test/test_linkedin_oauth.py

```
@@ -1,79 +1,78 @@
+
+def exchange_code_for_token(code):
+    url = "https://www.linkedin.com/oauth/v2/accessToken"
+    data = {
+        "grant_type": "authorization_code",
+        "code": code,
+        "redirect_uri": REDIRECT_URI,
+        "client_id": CLIENT_ID,
+        "client_secret": CLIENT_SECRET
+    }
+    headers = {"Content-Type": "application/x-www-form-urlencoded"}
+    resp = requests.post(url, data=data, headers=headers)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)